src/H5FDmulti.c

```diff
@@ -624,7 +624,7 @@ H5FD_multi_sb_encode(H5FD_t *_file, char *name /*out*/, unsigned char *buf /*out
     H5Eclear2(H5E_DEFAULT);
 
     /* Name and version number */
-    strncpy(name, "NCSAmulti", (size_t)8);
+    strncpy(name, "NCSAmult", (size_t)9);
```

Comment:
Previously was trying to copy 8 bytes out of a 10 byte (including NUL terminator) string?

Reply:
Nope  - there's only room for 8 bytes in the "Driver ID" field for the file format:  https://portal.hdfgroup.org/display/HDF5/File+Format+Specification

Reply:
I understand that. However,

1. "NCSAmulti" is 10 bytes long. Let's not confuse the issue by copying from a string that's larger than what we're actually using. Just format the string according to what we're actually going to use, e.g. "NCSAmult".
2. The header comment specifically says a 9-byte buffer is used: 8 bytes for the file format piece + a NUL terminator. Therefore, I want to copy at most 9 bytes from the source. I realize that we immediately manually NUL terminate the string, but I prefer to use strncpy how it's intended.